spec/models/api_wrappers/kraken_api_wrapper_spec.rb

```diff
@@ -201,9 +289,25 @@ def stub_transactions(count: 1, price: 1.5, amount: 2.5)
   end
 
   it '#find_lost' do
-    stub_private_client
     stub_orders
 
-    described_class.orders.all? { |o| described_class.find_lost(o.type, o.price, o.amount).present? }
+    api_wrapper.orders.all? { |o| api_wrapper.find_lost(o.type, o.price, o.amount).present? }
+  end
+
+  it '#currency_pair' do
+    stub_assets
+    BitexBot::Settings.taker.kraken.currency_pair.should eq taker_settings.kraken.currency_pair
+
+    api_wrapper.currency_pair.should be_a(HashWithIndifferentAccess)
+    api_wrapper.currency_pair.keys.should include(*%w[altname base quote raw_name])
+  end
+
+  it do
```

Comment:
We should look into better ways to test the settings, this seems a bit self-fulfilling.